### PR TITLE
feat: add ability to load capabilities from env vars

### PR DESCRIPTION
**Pull request by jonathannorris**

### CODE REVIEW

**Comment by JamieSinn** [comment]
Don't you want to load them from the `/spec` endpoint? https://github.com/DevCycleHQ/test-harness/blob/main/proxies/go/app.go#L34

File: harness/types/capabilities.test.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        const { SDKCapabilities } = jest.requireActual('../capabilities')
+        expect(SDKCapabilities).toEqual({
+            NodeJS: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+            Python: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+        })
+    })
+
+    it('should load capabilities from env variables as JSON', () => {
+        process.env.SDK_CAPABILITIES =
+            '["edgeDB","cloud","clientCustomData","sse"]'
```
Comment (ajwootto) [inline harness/types/capabilities.test.ts:20]: why bother supporting both formats?

File: harness/types/capabilities.test.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        process.env.SDKS_TO_TEST = '["NodeJS","Python"]'
+        const { SDKCapabilities } = jest.requireActual('../capabilities')
+        expect(SDKCapabilities).toEqual({
+            NodeJS: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+            Python: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+        })
+    })
+
+    it('should use default capabilities if env variables are not set', () => {
+        process.env.SDK_CAPABILITIES = 'edgeDB,cloud,clientCustomData,sse'
```
Comment (ajwootto) [inline harness/types/capabilities.test.ts:30]: shouldnt this not be set then?

File: harness/types/capabilities.test.ts (hunk 1 above)
Comment (jonathannorris) [reply harness/types/capabilities.test.ts:20]: basically copied this / wanted to match the `getSDKs()` method: 
image

File: harness/types/capabilities.test.ts (hunk 2 above)
Comment (jonathannorris) [reply harness/types/capabilities.test.ts:30]: The idea is for this to only be used when `SDKS_TO_TEST` is set. The idea is for the various SDKs to be able to set their capabilities when making breaking changes, but didn't want to turn both the existing `SDKS_TO_TEST` + `SDK_CAPABILITIES` into a single new JSON map you need to pass in. Seeing that we are only really going to use this for a single repo testing a new capability I thought this trade-off was fine.

Comment (jonathannorris) [reply harness/types/capabilities.test.ts:30]: so this test is testing that basically `SDK_CAPABILITIES` isn't used when `SDKS_TO_TEST` isn't set.

**Review by ajwootto — APPROVED** [review]

**Pull request merged**